packages/metablocks/src/accounts/utils.ts

```diff
@@ -79,11 +79,29 @@ const getAllAccountInfo = async <T>(
     }
   );
 
+  return processAccountInfo(resp, layouts);
+};
+
+const processAccountInfo = (
+  resp: {
+    pubkey: anchor.web3.PublicKey;
+    account: anchor.web3.AccountInfo<Buffer>;
+  }[],
+  layouts: Layout[]
+) => {
   return resp.map(({ pubkey, account }) => {
-    return {
-      publicKey: pubkey,
-      account: layout.decode(account.data) as T,
-    };
+    let result = null;
+    for (let layout of layouts) {
```

Comment:
`for`-> `map`?

Reply:
This couldn't be used as map because we will not be able to use `try catch` block